src/main/java/org/opentestsystem/ap/ims/service/impl/ItemEntityServiceImpl.java

```diff
@@ -73,6 +78,15 @@ public void initialize() {
                 log.error(format("Error creating folder %s.  Message: %s", path, e.getMessage()), e);
             }
         }
+
+        this.customMediaTypes = new HashMap<>();
+
+        this.customMediaTypes = this.contentTypeMappings.getMappings().entrySet().stream()
+            .collect(
+                Collectors.toMap(e ->
+                    Pattern.compile(".*[.]" + e.getKey()), e -> org.apache.tika.mime.MediaType.parse(e.getValue())
```

Comment:
Here is the conversion from the properties in the yaml to the actual Tika map.  Tika uses a Pattern as the map key and the content type value as the map value.